server/sonar-web/src/main/js/apps/issues/workspace-list-item-view.js

```diff
@@ -127,7 +127,7 @@ export default IssueView.extend({
       index,
       selected
     }, { silent: true });
-    return this.model.fetch(options).done(function () {
+    return this.model.fetch(options).done(() => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)